lib/WebService/Stripe.pm

```diff
@@ -54,6 +54,14 @@ method create_customer(Maybe[HashRef] $data, :$headers) {
     return $self->post( "/v1/customers", $data, headers => $headers );
 }
 
+method create_recipient(Maybe[HashRef] $data, :$headers) {
```

Comment:
:100: I was wondering about that, just copying convention from `Customer`, but that's prob a special case since we make them with no args.